src/tools/perf/lib/ucp_tests.cc

```diff
@@ -880,6 +913,7 @@ class ucp_perf_test_runner {
 
             wait_last_iter(recv_buffer, send_length);
             wait_recv_window(m_max_outstanding);
+            send_ack(ep, send_buffer, send_datatype, remote_addr, rkey);
```

Comment:
maybe send_ack/recv_ack could take ep, remote_addr, rkey from m_perf instead of passing them as params?

Reply:
ok